README.md

```diff
@@ -68,6 +66,4 @@ Create a branch for your work in the repo:
 
 1. Stage and commit your changes (`git add -A && git commit -m "Describe what you did"`)
 1. Push your changes (`git push origin your-branch-name`)
-1. [Create a Pull Request](https://help.github.com/articles/creating-a-pull-request/) against the `master` branch. Netlify will publish a [deploy preview](https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/) so you can share and preview your changes as a live site.
-
-After you push your changes, you may see a commit you didn't make such as `your-branch-name npm audit fix`. We use [auditmated](https://github.com/SparkPost/auditmated) to help keep our dependencies secure and up-to-date. To skip this, use the `--no-verify` command line option when you push your code.
+1. [Create a Pull Request](https://help.github.com/articles/creating-a-pull-request/) against the `main` branch. Netlify will publish a [deploy preview](https://www.netlify.com/blog/2016/07/20/introducing-deploy-previews-in-netlify/) so you can share and preview your changes as a live site.
```

Comment:
I dont think this actually happens?

Reply:
it should - it's one of the last steps. here's my preview - https://deploy-preview-487--developers-sparkpost.netlify.app/